docs/source/plugin/bot.rst

```diff
@@ -6,426 +6,8 @@ Once a :term:`Rule` has been triggered, it's time to do whatever the plugin is
 supposed to do. Thanks to the ``bot`` parameter, you can make the bot talk:
 say something in a channel, reply to someone, send a notice, or join a channel.
```

Comment:
I think it would be better to be a bit more generic about making the bot do stuff here, now that the page has been split into subpages with the TOC tree shown here.

```suggestion
supposed to do. Thanks to the ``bot`` parameter, you can make the bot say or do things:
send a message, reply to the person who triggered it, send a notice, or join a channel.
```

(Not bothering with re-wrapping just yet, as this is only a rough suggestion.)